src/IRremoteESP8266.h

```diff
@@ -1,1051 +1,1062 @@
- /***************************************************
```

Comment:
Something is broken causing the `diff` to highlight the entire file. This needs to be fixed so that it only shows us the bits that have changed. Check the line ending settings for your editor. That may be incorrect. It should be the same as the existing file. e.g. Lines are ended with a `LF` not a `CR+LF`.
https://en.wikipedia.org/wiki/Newline

(Assuming that is the problem)